README.md

```diff
@@ -88,5 +88,26 @@ port: 80
 $ amcrest-cli --camera living_room --version-http-api
 version=1.40
 ```
+
+- TUI
+```bash
+Configure amcrest.conf and trigger amcrest-tui, make sure the user
+triggering amcrest-tui have access to framebuffer device or use sudo.
```

Comment:
```bash
↪ ./amcrest-tui 
Traceback (most recent call last):
  File "./amcrest-tui", line 188, in <module>
    with open(args.framebuffer_device, 'r') as f:
PermissionError: [Errno 13] Permission denied: '/dev/fb0'
```
I see that you added this on the documentation, but let's add the command to allow the user to have access to /dev/fb0

```bash
↪ ls -la /dev/fb0
crw-rw----. 1 root video 29, 0 Jan 31 01:03 /dev/fb0
↪ sudo usermod  -a -G video <user>
```